tests/test_interfaces.py

```diff
@@ -30,7 +32,8 @@ def converter():
     converter = DocumentConverter(
         format_options={
             InputFormat.PDF: PdfFormatOption(
-                pipeline_options=pipeline_options, backend=DoclingParseDocumentBackend
+                pipeline_options=pipeline_options,
+                backend=DoclingParseDocumentBackend,
```

Comment:
I don't think this should be set at all

Reply:
For now it has to be set everywhere on tests that can generate GT.

Reply:
It is removed here.

Reply:
the purpose of this test unit is to check calling with streams or path. I think we should drop backend here AND remove the GENERATE flag as well.
(but it could be done in the next PR)